docs/overview/getting-started/avalanche-platform.md

```diff
@@ -4,40 +4,48 @@ description: Avalanche features 3 built-in blockchains which includes Exchange C
-also be a member of the Primary Network by staking at least 2,000 AVAX.
+Avalanche is a heterogeneous network of blockchains. As opposed to homogeneous networks, where
+all applications reside in the same chain, heterogeneous networks allow separate chains to be
+created for different applications.
+
+The Primary Network is a special [Subnet](../../subnets/README.md) that contains all validators
+(including validators of any custom Subnets). A node can become a validator for the Primary
+Network by staking at least 2,000 AVAX.
 
 ![Primary network](/img/primary-network.png)
 
+All validators as members of the Primary Network are required to validate and secure the
+following 3 blockchains:
+the [**Exchange Chain (X-Chain)**](#exchange-chain-x-chain),
+the [**Platform Chain (P-Chain)**](#platform-chain-p-chain), and
+the [**Contract Chain (C-Chain**)](#contract-chain-c-chain).
+
 ## Contract Chain (C-Chain)
 
-The **C-Chain** allows for the creation smart contracts using the [C-Chain’s API](../../apis/avalanchego/apis/c-chain.md).
+The **C-Chain** is an implementation of the Ethereum Virtual Machine (EVM).
+The [C-Chain’s API](../../apis/avalanchego/apis/c-chain.md) supports Geth's API and supports the
+creation and execution of smart contracts written in Solidity.
 
-The C-Chain is an instance of the Ethereum Virtual Machine powered by [Avalanche](./intro.md).
+The C-Chain is an instance of the Coreth Virtual Machine.
 
 ## Platform Chain (P-Chain)
 
-The **P-Chain** is the metadata blockchain on Avalanche and coordinates validators, keeps track of active
-Subnets, and enables the creation of new Subnets. The P-Chain implements 
-the [Snowman consensus protocol](../../#snowman-consensus-protocol).
+The **P-Chain** is responsible for all validator and Subnet-level operations.
+The [P-Chain API](../../apis/avalanchego/apis/p-chain.md) supports the creation of new
+blockchains and Subnets, the addition of validators to Subnets, staking operations, and other
+platform-level operations.
 
-The [P-Chain API](../../apis/avalanchego/apis/p-chain.md) allows clients to create Subnets, add validators
-to Subnets, and create blockchains.
+The P-Chain is an instance of the Platform Virtual Machine.
 
 ## Exchange Chain (X-Chain)
 
-The **X-Chain** acts as a decentralized platform for creating and trading digital smart assets, a
-representation of a real-world resource (for example, equity, bonds) with a set of rules that 
-govern its behavior, like "can’t be traded until tomorrow" or "can only be sent to US citizens."
+The **X-Chain** is responsible for operations on digital smart assets known as **Avalanche Native
+Assets**. A smart asset is a representation of a real-world resource (for example, equity, or a
+bond) with sets of rules that govern its behavior, like "can’t be traded until tomorrow" or "can
+only be sent to US citizens." The [X-Chain API](../../apis/avalanchego/apis/x-chain.md) supports the
+creation and trade of Avalanche Native Tokens.
```

Comment:
Can we remove "can only be sent to US citizens". I'm not sure who wrote this, but that's not true.

Reply:
Yeah this was copy pasta'd from old content. I can remove this.